docs/source/daml/intro/7_Composing.rst

```diff
@@ -0,0 +1,271 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+7 Composing Choices
+===================
+
+It is time to put everything you have learnt so far together into a complete and secure DAML model for asset issuance, management, transfer, and trading, with capabilities similar to that presented in :doc:`/getting-started/quickstart`. In the process you will learn about a few more concepts:
+
+- DAML Projects, Packages and Modules
+- Composition of transactions
+- Observers and Stakeholders
+- DAML's execution model
+- Privacy
+
+The model in this section is not a single DAML file, but a DAML project consisting of several files that depend on each other.
+
+DAML Projects
+-------------
+
+DAML is organized in packages and modules. DAML projects are specified using a ``daml.yaml`` file and compile into packages. DAML files within a project become DAML modules. You can start a new project with a skeleton structure using ``daml new project_name`` in the terminal.
+
+Each DAML project has a main source file, which is the entry point for the compiler. A common pattern is to have a main file called ``LibraryModules.daml``, which simply lists all the other modules to include.
+
+A minimal project would contain just two files: ``daml.yaml`` and ``daml/LibraryModules.daml``. Take a look at the ``daml.yaml`` for this project:
+
+.. literalinclude:: daml/daml-intro-7/daml.yaml.template
+  :language: yaml
+
+You can generally set ``name`` and ``version`` freely to describe your project. ``dependencies`` lists package dependencies. ``daml-prim`` should always be included. ``daml-stdlib`` gives access to the DAML standard library.
+
+You can compile a DAML project by running ``daml build`` from the project root directory. This creates a ``dar`` package in ``dist/project_name.dar``. A ``dar`` file is DAML's equivalent of a ``JAR`` file in Java. It's the artifact that gets deployed to a ledger to load the contract model.
```

Comment:
Bit confusing to have it called both a dar "file" and "package" - maybe feels more correct to say that a dar file is a package/DAML package? Not sure